tests/unit/views/Player.spec.ts

```diff
@@ -0,0 +1,7 @@
+// NOTE --
+// Unit tests for Player.vue have been skipped becauase
+// the E2E tests for Player.vue cover the unit tests functionality as well
+
+describe('DummyTestSuite', () => {
+  it('DummyTest', () => {})
+})
```

Comment:
why do we need this then?

Reply:
I spent a whole day writing unit tests for player because i didn't see `Player.spec.js` file and I didn't know that its unit tests were being covered in E2E tests, it's not written anywhere as well.
Writing it here so in future someone doesn't waste their entire day like me because they didn't see a file sitting there :p

Reply:
Just saw this comment randomly. Your time was not wasted!

Purpose of unit tests and E2E tests are very different, and it's okay if they cover the same ground!